redux/main.py

```diff
@@ -91,11 +91,12 @@ def __init__(
         self._event_handlers_queue = queue.Queue[
             tuple[EventHandler[Event], Event] | None
         ]()
-        self_workers = [
+        self._workers = [
```

Comment:
Nice catch! But since this isn't needed in any methods, we can consider it a fortunate typo. Let's rename it to `workers` and keep it as a local variable.